pkg/cmd/server/admin/default_certs.go

```diff
@@ -187,36 +188,34 @@ func DefaultNodeDir(nodeName string) string {
 	return "node-" + nodeName
 }
 
-func DefaultNodeServingCertInfo(certDir, nodeName string) configapi.CertInfo {
+func DefaultNodeServingCertInfo(nodeDir string) configapi.CertInfo {
 	return configapi.CertInfo{
-		CertFile: path.Join(certDir, DefaultNodeDir(nodeName), "server.crt"),
-		KeyFile:  path.Join(certDir, DefaultNodeDir(nodeName), "server.key"),
+		CertFile: path.Join(nodeDir, "server.crt"),
+		KeyFile:  path.Join(nodeDir, "server.key"),
 	}
 }
-func DefaultNodeClientCertInfo(certDir, nodeName string) configapi.CertInfo {
+func DefaultNodeClientCertInfo(nodeDir string) configapi.CertInfo {
 	return configapi.CertInfo{
-		CertFile: path.Join(certDir, DefaultNodeDir(nodeName), "master-client.crt"),
-		KeyFile:  path.Join(certDir, DefaultNodeDir(nodeName), "master-client.key"),
+		CertFile: path.Join(nodeDir, "master-client.crt"),
+		KeyFile:  path.Join(nodeDir, "master-client.key"),
 	}
 }
-func DefaultNodeKubeConfigFile(certDir, nodeName string) string {
-	return path.Join(certDir, DefaultNodeDir(nodeName), ".kubeconfig")
+func DefaultNodeKubeConfigFile(nodeDir string) string {
+	return path.Join(nodeDir, ".kubeconfig")
```

Comment:
> no dotfiles, right?

It's a kubeconfig file used for pointing to a kube server.  There's a reasonable expectation that it will be used by kubectl, which expects `.kubeconfig`